### PR TITLE
Transposed Convolution is not a deconvolution

### PR DESCRIPTION
**Pull request by lucasSaavedra123**
It is widely used the word "Deconvolution" when we refer to Transposed Convolution. However, that's wrong and in the code documentation this is not clarified. I added a simple clarification of it. Maybe a short explanation is needed.

### CODE REVIEW

**Comment by fchollet** [comment]
Thanks for the PR.

It is sometimes called deconvolution. That statement is accurate. The term itself may be an inaccurate, but the Keras docs are not there to issue value judgements about mainstream terminology usage. Otherwise we'd have to call `BatchNormalization` `BatchStandardization` instead, etc.

**Comment by lucasSaavedra123** [comment]
Yep, you're right.
